sdk/appconfiguration/azure-appconfiguration/README.md

```diff
@@ -9,7 +9,7 @@ Use the client library for App Configuration to create and manage application co
 [Source code](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appconfiguration/azure-appconfiguration)
 | [Package (Pypi)][package]
 | [Package (Conda)](https://anaconda.org/microsoft/azure-appconfiguration/)
-| [API reference documentation](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appconfiguration/azure-appconfiguration)
+| [API reference documentation](https://learn.microsoft.com/python/api/overview/azure/appconfiguration-readme?view=azure-python)
```

Comment:
https://learn.microsoft.com/en-us/python/api/azure-appconfiguration/azure.appconfiguration?view=azure-python

Reply:
locale is not allowed here, the link in my commit can redirect to the same page as with locale.

Reply:
Follow what python repo does.